components/brave_wallet_ui/common/async/handlers.ts

```diff
@@ -150,6 +152,14 @@ handler.on(WalletActions.defaultWalletChanged.getType(), async (store) => {
   await refreshWalletInfo(store)
 })
 
+handler.on(WalletActions.defaultBaseCurrencyChanged.getType(), async (store, payload: DefaultBaseCurrencyChanged) => {
+  console.log('base currency changed----', payload)
```

Comment:
nit: Remove this debug log maybe? Probably fine to keep it for now if UI change is going to land pretty soon.